scripts/admincommands.js

```diff
@@ -147,22 +147,22 @@ exports.handleCommand = function(src, command, commandData, tar, channel) {
         var regex;
         try {
             regex = new RegExp(commandData.toLowerCase()); // incase sensitive
-        } catch (e) {

```

Comment:
I thought we using "error" now? Thought I recall Sulcuta changing "e" and "err" to "error" on battlefactory. Plus e can be confused with element and error. When linting I was getting complants that using e was bad for for-in loops which I presume was that reason.